drivers/shared/executor/executor_linux.go

```diff
@@ -519,10 +519,9 @@ func configureIsolation(cfg *lconfigs.Config, command *ExecCommand) error {
 
 	cfg.Mounts = []*lconfigs.Mount{
 		{
-			Source:      "/dev",
+			Source:      "dev",
 			Destination: "/dev",
-			Device:      "bind",
-			Flags:       syscall.MS_BIND | syscall.MS_RDONLY | syscall.MS_NOEXEC,
```

Comment:
Noticed that 0.8 was mounting `/dev` with `MS_RDONLY` in [1] - but this prevents any additional dev mknod operation by libcontainer (for gpu support), so left it as read-write mount.

[1] https://github.com/hashicorp/nomad/blob/acae110c7bf3ebafbb364fd0e4af0a2324e15395/client/allocdir/task_dir_linux.go#L27